src/main/duke/TaskList.java

```diff
@@ -0,0 +1,33 @@
+package main.duke;
+
+import main.duke.tasks.Task;
+
+import java.util.ArrayList;
+
+public class TaskList {
+    private ArrayList<Task> Tasks;
```

Comment:
I think the variable name should be in camelCase. I suggest it should be _tasks_